CHANGELOG.md

```diff
@@ -6,6 +6,64 @@ A breaking change will get clearly marked in this log.
 
 ## Unreleased
 
+### Breaking Changes
+
+- `ContractClient` functionality previously added in [v11.3.0](https://github.com/stellar/js-stellar-sdk/releases/tag/v11.3.0) was exported in a non-standard way. You can now import it as any other stellar-sdk module.
+
+  ```diff
+  - import { ContractClient } from '@stellar/stellar-sdk/lib/contract_client'
+  - import { ContractClient } from '@stellar/stellar-sdk'
+  ```
+
+  Note that this top-level export is now a container for all ContractClient-related functionality. The ContractClient class is now available at `ContractClient.Client`.
+
+  Additionally, it is available from the `/ContractClient` entrypoint, if your version of Node ([and TypeScript!](https://stackoverflow.com/a/70020984/249801)) supports [the `exports` declaration](https://nodejs.org/api/packages.html#exports). Finally, some of its exports have been renamed.
+
+  ```diff
+   import {
+  -  ContractClient,
+  +  Client,
+     AssembledTransaction,
+  -  ContractClientOptions,
+  +  Options,
+     SentTransaction,
+  -} from '@stellar/stellar-sdk/lib/contract_client'
+  +} from '@stellar/stellar-sdk/ContractClient'
+  ```
+
+
+- `ContractSpec` class now nested under `ContractClient` module, and has been renamed to `Spec`.
+
+  ```diff
+  -import { ContractSpec } from '@stellar/stellar-sdk'
+  +import { ContractClient } from '@stellar/stellar-sdk'
+  +const { Spec } = ContractClient
+  ```
+
+  Alternatively, you can import this from the `ContractClient` entrypoint.
+
+  ```diff
+  -import { ContractSpec } from '@stellar/stellar-sdk'
+  +import { Spec } from '@stellar/stellar-sdk/ContractClient'
+  ```
+
+### Added
+
+- `SorobanRpc` module is now also exported as `Rpc`. You can import it with either name.
```

Comment:
Nice! I'm for this; can you also note that `SorobanRpc` is now deprecated and will be removed in the future?

Reply:
Are we ready to officially deprecate it? Does that require anything more than mentioning it in the changelog?

Reply:
Just the latter, it'll be informal

Reply:
Ok, changelog mentions that it's deprecated.